website/pages/api-docs/secret/openldap/index.mdx

```diff
@@ -33,6 +33,8 @@ to search and change entry passwords in OpenLDAP.
 - `binddn` `(string: <required>)` - Distinguished name (DN) of object to bind for managing user entries. Example: `cn=vault,ou=Users,dc=hashicorp,dc=com`
 - `bindpass` `(string: <required>)` - Password to use along with `binddn` for managing user entries.
 - `url` `(string: <optional>)` - The LDAP server to connect to. Examples: `ldaps://ldap.myorg.com`, `ldaps://ldap.myorg.com:636`. This can also be a comma-delineated list of URLs, e.g. `ldaps://ldap.myorg.com,ldaps://ldap.myorg.com:636`, in which case the servers will be tried in-order if there are errors during the connection process. Default is `ldap://127.0.0.1`.
+- `length` `(int: 64)` - The length of generated password strings.
```

Comment:
Consider adding a generic caveat like, "Note: some schemas may require shorter passwords" or something like that.

Reply:
Done!